test/test_foreach.py

```diff
@@ -1432,6 +1433,10 @@ def check_autodiff_sample(op, sample, dtype, is_inplace):
             "Trying to set a forward gradient that has a different size than that of the original Tensor, "
             "this is not supported. Tensor is of size [] while the given forward gradient is of size [1, 1].",
         )
+    if op.name == "_foreach_rsqrt" and dtype == torch.float64 and not is_inplace:
```

Comment:
We've been avoiding the zero size inputs in the sample inputs generation, if the foreach op is not going to run with 0sized inputs.: https://github.com/pytorch/pytorch/blob/92e38a476fc28a18c962dff3a6b0d49dfc8ef8da/torch/testing/_internal/common_methods_invocations.py#L9533

Reply:
It'd be better to not generate them in the first place if we're going to skip them later. Unless there's a test for rsqrt that can take in zerodim tensors?

Reply:
I think zero size samples are generated based on the value of `intersperse_empty_tensors` around https://github.com/pytorch/pytorch/blob/4fcd15a667df5b80e81db6563d8d3123a0cbd051/torch/testing/_internal/common_methods_invocations.py#L9487

Reply:
https://github.com/pytorch/pytorch/blob/4fcd15a667df5b80e81db6563d8d3123a0cbd051/torch/testing/_internal/common_methods_invocations.py#L9531

Haha I think rsqrt could follow the example in foreach_max then if it suffers the same consequences.

Reply:
I'm confused where rsqrt calls max though...

Reply:
locally it worked without this check. I might have been confused by the combination of zero-size tensors and zeroes in tensors.